components/pricing_modal/content.tsx

```diff
@@ -181,6 +190,34 @@ function Content(props: ContentProps) {
         }));
     };
 
+    const downgrade = async () => {
+        if (!starterProduct) {
+            return;
+        }
+
+        const result = await dispatch(subscribeCloudSubscription(starterProduct?.id));
+
+        if (typeof result === 'boolean') {
```

Comment:
Nit: Even though this action currently only returns a true as a boolean, could we do a `if (typeof result === 'boolean') && result) {` here to future proof against if an additional error state that returns `false` is added?